packages/core/botpress/src/cli/init.js

```diff
@@ -95,6 +95,11 @@ const generate = async result => {
   const info = JSON.parse(files['info.json'])
   delete files['info.json']
 
+  // Npm renames .gitignore into .npmignore while publishing package which we need to revert
+  // See https://github.com/npm/npm/wiki/Files-and-Ignores#details-1 for more details
+  files['.gitignore'] = files['.npmignore']
```

Comment:
Let's actually wrap this inside a if (`if files['.npmignore']`), which would bulletproof this solution if that ever changes